generate.go

```diff
@@ -646,8 +648,53 @@ func getDefaultTemplate() specs.Spec {
 			},
 		},
 		Hostname: "shell",
-		Mounts:   []specs.Mount{},
+		Mounts: []specs.Mount{
+			{
+				Destination: "/proc",
+				Type:        "proc",
+				Source:      "proc",
+				Options:     nil,
+			},
+			{
+				Destination: "/dev",
+				Type:        "tmpfs",
+				Source:      "tmpfs",
+				Options:     []string{"nosuid", "strictatime", "mode=755", "size=65536k"},
+			},
+			{
+				Destination: "/dev/pts",
```

Comment:
This one is causing trouble in my ccon-oci tests.  The previous mount *just* put a tmpfs at /dev, so there is no directory at `/dev/pts` to attach the mount.  Testing with a few patches to get ccon-oci and `runtime_test.sh` working together:

    $ ./test_runtime.sh -r 'ccon-oci --verbose'
    -----------------------------------------------------------------------------------
    VALIDATING RUNTIME: ccon-oci --verbose
    -----------------------------------------------------------------------------------
    launched container process with PID 22796
    write '0 1000 1' to /proc/22796/uid_map
    write 'deny' to /proc/22796/setgroups
    write '0 1000 1' to /proc/22796/gid_map
    mount 0: /tmp/tmp.MZQVJfex0V/busybox/./rootfs to /tmp/tmp.MZQVJfex0V/busybox/./rootfs (type: (null), flags: 4096, data (null))
    mount 1: /tmp/tmp.MZQVJfex0V/busybox/proc to /tmp/tmp.MZQVJfex0V/busybox/rootfs/proc (type: proc, flags: 0, data (null))
    mount 2: /tmp/tmp.MZQVJfex0V/busybox/tmpfs to /tmp/tmp.MZQVJfex0V/busybox/rootfs/dev (type: tmpfs, flags: 16777218, data mode=755,size=65536k)
    mount 3: /tmp/tmp.MZQVJfex0V/busybox/devpts to /tmp/tmp.MZQVJfex0V/busybox/rootfs/dev/pts (type: devpts, flags: 10, data newinstance,ptmxmode=0666,mode=0620,gid=5)
    mount: No such file or directory
    …

I don't see any language about “the runtime MUST create missing destinations” [in the spec][1], and I don't see a way to support this default template *without* doing that.  Are there suggestions for how other runtimes should handle entries like this until [runC sorts out][2] its [required filesystems][3] support?

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/config.md#mounts
[2]: https://github.com/opencontainers/ocitools/pull/24/files#r57248068
[3]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc1/config-linux.md#default-file-systems